schema/workflow.yaml

```diff
@@ -5,6 +5,7 @@ type: object
 properties:
   document:
     type: object
+    unevaluatedProperties: false
```

Comment:
I would assume `document` could be extended. Some runtimes could use it to store metadata for instance. (e.g. https://cloud-native.slack.com/archives/CMBAZQSEP/p1721730364697849 )

Reply:
@JBBianchi Spot on, totally agree with you!